### PR TITLE
vertexai: Include avg_logprobs in response metadata

### LINKED ISSUE #525: Add support for AvgLogprobs to VertexAI integrations
Hi LangChain Team!

Can we please add support for returning AvgLogprobs from calls to VertexAI models please?

Thank you!

Regards
Jasen

### PR DESCRIPTION
**Pull request by jzaldi**
## PR Description

Includes `avg_logprobs` in the response metadata

## Relevant issues

Fixes #525

## Type

🆕 New Feature

### CODE REVIEW

File: libs/vertexai/langchain_google_vertexai/_utils.py [hunk 1]
```diff
@@ -176,6 +176,7 @@ def get_generation_info(
             "finish_reason": (
                 candidate.finish_reason.name if candidate.finish_reason else None
             ),
+            "avg_logprobs": candidate.avg_logprobs,
```
Comment (lkuligin) [inline libs/vertexai/langchain_google_vertexai/_utils.py:179]: is it exposed for all models now? should we check whether the attribute exists maybe?

Comment (jzaldi) [reply libs/vertexai/langchain_google_vertexai/_utils.py:179]: It's just for Gemini, is an attribute of the class of the candidates so idk we can check if the attribute exists or maybe doesn't include it if it's None

Comment (lkuligin) [reply libs/vertexai/langchain_google_vertexai/_utils.py:179]: yep, like at the line above, plaese

Comment (jzaldi) [reply libs/vertexai/langchain_google_vertexai/_utils.py]: Added the checks

**Review by lkuligin — APPROVED** [review]

**Comment by jaymon0703** [comment]
Thank you @lkuligin and @jzaldi

**Pull request merged**